### PR TITLE
Add image srcset to large images and optimise image sizes

### LINKED ISSUE #407: Optimize Image Assets
This would be all app images, NOT project images

- What's the Vue approach to optimizing images?
- What size should hero images be?
- Homepage is the main priority here

### PR DESCRIPTION
**Pull request by minimalsm**
### Description

- Adds srcset to all hero/large images on the site so that we aren't loading larger than necessary images
- Adds optimised images (360, 720, 1080, 1440, 2160 and 2880px versions of these images)
- Moves main images from `/assets/` to `/assets/imageName`

### Additional

- was only able to test this on pages that before joining the round
- could refactor this into a component as all images share the same breakpoints?

### Reference

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#using_the_srcset_and_sizes_attributes

Fixes: https://github.com/ethereum/clrfund/issues/407

### CODE REVIEW

File: vue-app/src/components/RoundInformation.vue [hunk 1]
```diff
@@ -2,7 +2,19 @@
   <div class="projects">
     <div class="round-info">
       <div class="image-wrapper">
-        <img src="@/assets/docking.png" height="100%" />
+        <img
+          src="@/assets/docking/docking.png"
```
Comment (pettinarip) [inline vue-app/src/components/RoundInformation.vue]: We would need the original images to be optimized too in case the browser do not support these attrs.

**Review by pettinarip — APPROVED** [review]
Great job @minimalsm! thanks for doing this.

I would like to see that refactor that you mentioned 👍🏼

Also, this is related to the @corwintines PR #410 maybe you can coordinate to avoid conflicts.

File: vue-app/src/components/RoundInformation.vue (hunk 1 above)
Comment (corwintines) [reply vue-app/src/components/RoundInformation.vue]: I agree, this is great but maybe a more optimized image for the default. Ill make some code suggestions, but other than that this is great! Ill default to the 1080 one.

Comment (corwintines) [reply vue-app/src/components/RoundInformation.vue]:
```suggestion
          src="../assets/docking/docking_w1080.png"
```

File: vue-app/src/views/JoinLanding.vue [hunk 2]
```diff
@@ -2,7 +2,18 @@
   <div>
     <div class="gradient">
       <div class="hero">
-        <img src="@/assets/core.png" />
+        <img
+          src="@/assets/core/core.png"
```
Comment (corwintines) [inline vue-app/src/views/JoinLanding.vue]:
```suggestion
          src="@/assets/core/core_w1080.png"
```

File: vue-app/src/views/Landing.vue [hunk 3]
```diff
@@ -5,7 +5,18 @@
       <div id="hero">
         <img src="@/assets/moon.png" id="moon" />
         <div class="image-wrapper">
-          <img src="@/assets/docking.png" />
+          <img
+            src="@/assets/docking/docking.png"
```
Comment (corwintines) [inline vue-app/src/views/Landing.vue]:
```suggestion
            src="@/assets/docking/docking_w1080.png"
```

File: vue-app/src/views/Landing.vue [hunk 4]
```diff
@@ -49,9 +60,18 @@
       <div id="section-how-it-works">
         <div class="wormhole-wrapper desktop-l">
           <img
-            src="@/assets/wormhole.png"
-            alt="Image of spaceships funneling through a wormhole and getting bigger"
+            src="@/assets/wormhole/wormhole.png"
```
Comment (corwintines) [inline vue-app/src/views/Landing.vue]:
```suggestion
            src="@/assets/wormhole/wormhole_w1080.png"
```

File: vue-app/src/views/Landing.vue [hunk 5]
```diff
@@ -61,7 +81,16 @@
             just those with the wealthiest backers.
           </p>
           <img
-            src="@/assets/wormhole.png"
+            src="@/assets/wormhole/wormhole.png"
```
Comment (corwintines) [inline vue-app/src/views/Landing.vue]:
```suggestion
            src="@/assets/wormhole/wormhole_w1080.png"
```

File: vue-app/src/views/ProjectAdded.vue [hunk 6]
```diff
@@ -4,7 +4,18 @@
     <div class="gradient">
       <img src="@/assets/moon.png" class="moon" />
       <div class="hero">
-        <img src="@/assets/newrings.png" />
+        <img
+          src="@/assets/newrings/newrings.png"
```
Comment (corwintines) [inline vue-app/src/views/ProjectAdded.vue]:
```suggestion
          src="@/assets/newrings/newrings_w1080.png"
```

File: vue-app/src/views/TransactionSuccess.vue [hunk 7]
```diff
@@ -1,7 +1,18 @@
 <template>
   <div class="container">
     <div class="image-wrapper">
-      <img class="docking" src="@/assets/docking.png" />
+      <img
+        src="@/assets/docking/docking.png"
```
Comment (corwintines) [inline vue-app/src/views/TransactionSuccess.vue]:
```suggestion
        src="@/assets/docking/docking_w1080.png"
```

File: vue-app/src/views/Verified.vue [hunk 8]
```diff
@@ -6,7 +6,18 @@
     <div class="gradient">
       <img src="@/assets/moon.png" class="moon" />
       <div class="hero">
-        <img src="@/assets/newrings.png" />
+        <img
+          src="@/assets/newrings/newrings.png"
```
Comment (corwintines) [inline vue-app/src/views/Verified.vue]:
```suggestion
          src="@/assets/newrings/newrings_w1080.png"
```

File: vue-app/src/views/VerifyLanding.vue [hunk 9]
```diff
@@ -7,7 +7,18 @@
       <div class="gradient">
         <img src="@/assets/moon.png" class="moon" />
         <div class="hero">
-          <img src="@/assets/newrings.png" />
+          <img
+            src="@/assets/newrings/newrings.png"
```
Comment (corwintines) [inline vue-app/src/views/VerifyLanding.vue]:
```suggestion
            src="@/assets/newrings/newrings_w1080.png"
```

**Review by corwintines — COMMENTED** [review]
Suggested changes to more optimized image

**Comment by minimalsm** [comment]
Committed suggestions.

On the refactoring, I have never touched a Vue component so I wouldn't know where to start 😅  feel free to commit any changes to this you feel necessary 👍

**Review by corwintines — APPROVED** [review]

File: vue-app/src/views/Landing.vue [hunk 10]
```diff
@@ -61,7 +81,16 @@
             just those with the wealthiest backers.
           </p>
           <img
-            src="@/assets/wormhole.png"
+            src="@/assets/wormhole/wormhole_w1080.png"
+            sizes="(max-width: 1440px) 100vw, 1440px"
+            srcset="
+              ../assets/wormhole/wormhole_w360.png   360w,
+              ../assets/wormhole/wormhole_w720.png   720w,
+              ../assets/docking/docking_w1080.png   1080w,
```
Comment (pettinarip) [inline vue-app/src/views/Landing.vue]: This one should be `/wormhole_w1080`, right?

**Comment by pettinarip** [comment]
@minimalsm I've implemented a small refactor, creating a reusable component. Let me know if you have any question on that. Otherwise, I'll merge it on Monday.

**Pull request merged**